### DISCUSSION TITLE
希望能够获取并下载杜比音轨

### DISCUSSION THREAD

**Discussion by LzNOHentai** [the1812/Bilibili-Evolved#1593] [功能建议 / Ideas]
这个视频在APP上可以选择缓存杜比音轨
https://www.bilibili.com/video/BV1St4y1B7Jq

**Comment by Geeyun-JY3** [comment]
#1297
> 善用搜索(界面汉化截图见下方), 请不要开重复的主题!

重复：#1412，#1445，#1484
